src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -76,6 +91,9 @@ public static final long divisionUsingShift(int a, int b) {
     }
 
     public static final long divisionUsingLogs(int a, int b) {
+        if ( b == 0 ) {
+            throw new IllegalArgumentException("Cannot divide by zero");
+        }
```

Comment:
Checks if b is equal to 0, then if true, throw an `IllegalArgumentException` in divisionUsingLogs function